build/utils.ts

```diff
@@ -146,8 +146,8 @@ export function splitSections(rawHTML) {
 
   const iterable = [...($("#_body")[0] as cheerio.Element).childNodes];
   let c = 0;
-  iterable.forEach((child: cheerio.Element) => {
-    if (child.tagName === "h2") {
```

Comment:
I'm noticing a lot of changes like this throughout the code.  What was the reasoning behind these changes?

Reply:
The reason is that `childNodes` is of type `ChildNode[]` and `ChildNode = Text | Comment | ProcessingInstruction | Element | CDATA | Document`, of which only `Element` has a `tagName`.

Same holds for the `$.each()` callback, which might include non-`Element`s.